build/yamls/base/conf/antrea-agent.conf

```diff
@@ -120,3 +120,12 @@ featureGates:
 # Provide the address of Kubernetes apiserver, to override any value provided in kubeconfig or InClusterConfig.
 # Defaults to "". It must be a host string, a host:port pair, or a URL to the base of the apiserver.
 #kubeAPIServerOverride: ""
+
+# Comma-separated list of cipher suites.
```

Comment:
Maybe we could match the K8s documentation here and add "If omitted, the default Go cipher suites will be used." And even add this link: https://golang.org/pkg/crypto/tls/#CipherSuites

Reply:
Updated. But I prefer this link since this one includes possible cipher suite value: https://golang.org/pkg/crypto/tls/#pkg-constants